### PR TITLE
Update docs with tap firewall rules

### LINKED ISSUE #524: GKE private cluster documentation update
## Bug Report

### What is the issue?
The documentation for GKE private clusters needs updating to reflect tap port from the apiserver to the nodes.

### How can it be reproduced?
Create a GKE private cluster, install linkerd, and run `kubectl apiservices`.  You'll see that the AVAILABLE value is `False (FailedDiscoveryCheck)`.

Add the appropriate firewall rules, and run `kubectl apiservices` again after a bit, and then the AVAILABLE value is `True`, and all is good again.

### Logs, error output, etc

```
kubectl get apiservice/v1alpha1.tap.linkerd.io                                                                                                                                                                                                                                                                                                                                                                                    
NAME                      SERVICE               AVAILABLE                      AGE
v1alpha1.tap.linkerd.io   linkerd/linkerd-tap   False (FailedDiscoveryCheck)   97m
```

```
gcloud compute firewall-rules list | awk '/gke-to-linkerd/ {print $1" "$3" "$5" "$6}'

To show all fields of the firewall, please show in JSON format: --format=json
To show all fields in table format, please see the examples in --help.

gke-to-linkerd-proxy-injector-8443 INGRESS tcp:8443 False
gke-to-linkerd-tap-8088 INGRESS tcp:8088 False
gke-to-linkerd-tap-8089 INGRESS tcp:8089 False
```

```
kubectl get apiservice/v1alpha1.tap.linkerd.io
NAME                      SERVICE               AVAILABLE   AGE
v1alpha1.tap.linkerd.io   linkerd/linkerd-tap   True        98m
```


#### `linkerd check` output

```text
linkerd check
kubernetes-api
--------------
√ can initialize the client
√ can query the Kubernetes API

kubernetes-version
------------------
√ is running the minimum Kubernetes API version
√ is running the minimum kubectl version

linkerd-config
--------------
√ control plane Namespace exists
√ control plane ClusterRoles exist
√ control plane ClusterRoleBindings exist
√ control plane ServiceAccounts exist
√ control plane CustomResourceDefinitions exist
√ control plane MutatingWebhookConfigurations exist
√ control plane ValidatingWebhookConfigurations exist
√ control plane PodSecurityPolicies exist

linkerd-existence
-----------------
√ 'linkerd-config' config map exists
√ control plane replica sets are ready
√ no unschedulable pods
√ controller pod is running
√ can initialize the client
√ can query the control plane API

linkerd-api
-----------
√ control plane pods are ready
√ control plane self-check
√ [kubernetes] control plane can talk to Kubernetes
√ [prometheus] control plane can talk to Prometheus
√ no invalid service profiles

linkerd-version
---------------
√ can determine the latest version
√ cli is up-to-date

control-plane-version
---------------------
√ control plane is up-to-date
√ control plane and cli versions match

Status check results are √
```

### Environment

- Kubernetes Version: 1.14.3-gke.11
- Cluster Environment: GKE
- Host OS: Container Optimised
- Linkerd version: 2.5.0

### Possible solution
Add the gcloud firewall-rules create command(s) to the documentation for GKE private clusters.
https://linkerd.io/2/reference/cluster-configuration/#private-clusters

### Additional context

### PR DESCRIPTION
**Pull request by cpretzer**
Signed-off-by: Charles Pretzer <charles@buoyant.io>

fixes #524

### CODE REVIEW

File: linkerd.io/content/2/reference/cluster-configuration.md [hunk 1]
```diff
@@ -68,6 +68,15 @@ gcloud compute firewall-rules create gke-to-linkerd-proxy-injector-8443 \
   --priority 1000
 ```
 
+```bash
+gcloud compute firewall-rules create gke-to-linkerd-tap-8088-8089 \
+  --network "$NETWORK" \
+  --allow "tcp:8088-8089" \
```
Comment (grampelberg) [inline linkerd.io/content/2/reference/cluster-configuration.md]: Do we need the range? Seems like it should be a single port.

Comment (grampelberg) [reply linkerd.io/content/2/reference/cluster-configuration.md]: Is there any way to combine these both into a single command? Maybe `tcp:8443,tcp:8088`?

**Comment by cpretzer** [comment]
I looked at the Google docs to see if multiple ports for a single protocol would work, and it doesn't look like it. Initially, I wanted to combine the tap and proxy injector rules in to one.

Using the range appeared to be the Google best practice (tm) based on the docs.

I'll try to create a rule with multiple ports for a single protocol and see what happens.

**Comment by cpretzer** [comment]
@grampelberg it turns out multiple ports for a single protocol works. I've updated the pull request with a consolidated rule

**Comment by ihcsim** [comment]
This fixes #544.

**Pull request merged**